server/middleware/authorisationMiddleware.js

```diff
@@ -0,0 +1,11 @@
+const jwtDecode = require('jwt-decode')
+
+module.exports = (req, res, next) => {
+  if (res.locals && res.locals.user && res.locals.user.token) {
+    const roles = jwtDecode(res.locals.user.token).authorities
+    res.locals.user = { isReviewer: roles.includes('ROLE_USE_OF_FORCE_REVIEWER'), ...res.locals.user }
```

Comment:
could roles be null?

Reply:
I'm now handling this and have improved tests - generating a token rather than hardcoding them in there